tests/test_pytest_cov.py

```diff
@@ -8,7 +8,7 @@
 import py
 import pytest
 import virtualenv
-from process_tests import TestProcess
```

Comment:
this is just to suppress the warnings (mentioned in [the commit message](https://github.com/davidszotten/pytest-cov/commit/3dbb47a28999346abade322cfd9fbc0c2dcec77b), but i agree that's not very visible). would you prefer a code comment?